cts/cts-cli.in

```diff
@@ -2385,10 +2385,46 @@ class CrmDiffRegressionTest(RegressionTest):
     @property
     def tests(self):
         """A list of Test instances to be run as part of this regression test."""
+
+        old_file = f"{cts_cli_data}/crm_diff_old.xml"
+        new_file = f"{cts_cli_data}/crm_diff_new.xml"
+        patch_file = f"{cts_cli_data}/crm_diff_patchset.xml"
+
+        with open(f"{cts_cli_data}/crm_diff_old.xml", "r") as file:
+            old_str = f"'{file.read()}'"
+        with open(f"{cts_cli_data}/crm_diff_new.xml", "r") as file:
+            new_str = f"'{file.read()}'"
```

Comment:
I might be missing something here... why can't these two just be `file.read()`?

Reply:
I think I did this because of nested quotes, so that I wouldn't have to deal with them later when building the command lines. This contains whitespace and will be used as a CLI argument to `crm_diff`, so we need to treat it as a unit.

Could easily be done in the command lines themselves though. Not sure which I prefer now.